src/GovUk.Education.ExploreEducationStatistics.Notifier/PublicationNotifier.cs

```diff
@@ -173,8 +186,8 @@ public class PublicationNotifier
 
                     var values = new Dictionary<string, dynamic>
                     {
-                        {"publication_name", title},
-                        {"verification_link", baseUrl + id + "/verify-subscription/" + activationCode}
+                        { "publication_name", title },
```

Comment:
As `title` has changed from `string` to `string?` is this not giving a warning that `dynamic` should be `dynamic?`?

Reply:
There is a null check on line 140
```
            if (id == null || email == null || slug == null || title == null)
            {
                return new BadRequestObjectResult("Please pass a valid email & publication");
            }
```

So I think this is okay

Reply:
Ok I see now, thanks. Nothing to do with this PR but it seems like we don't need to use `HttpRequest` as the request type, we can use a custom type according to [Payload](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=csharp#payload). Then we might be able to get some built in validation rather than having to declare these as nullable. Worth trying in future I guess.